server/util/flagutil/types/autoflags/autoflags.go

```diff
@@ -2,52 +2,26 @@ package autoflags
 
 import (
 	"flag"
-	"log"
 	"net/url"
 	"reflect"
 	"time"
 
+	"github.com/buildbuddy-io/buildbuddy/server/util/log"
+
 	flagtypes "github.com/buildbuddy-io/buildbuddy/server/util/flagutil/types"
+	flagtags "github.com/buildbuddy-io/buildbuddy/server/util/flagutil/types/autoflags/tags"
 	flagyaml "github.com/buildbuddy-io/buildbuddy/server/util/flagutil/yaml"
 )
 
-type Taggable interface {
-	Tag(name string)
-}
-
-type secretTag struct{}
+type Taggable flagtags.Taggable
 
-func (_ *secretTag) Tag(name string) {
-	log.Fatalf(
-		"'secretTag.Taggable' was called for flag '%s', but is not meant to be "+
-			"called for 'secretTag'; tagging a flag as secret is instead handled by "+
-			"'New' or 'Var'.",
-		name,
-	)
-}
-
-var SecretTag = &secretTag{}
+var SecretTag = flagtags.SecretTag
 
-type deprecatedTag struct {
-	migrationPlan string
-}
-
-func (d *deprecatedTag) Tag(name string) {
-	log.Fatalf(
-		"'deprecatedTag.Taggable' was called for flag '%s', but is not meant to "+
-			"be called for 'deprecatedTag'; tagging a flag as deprecated is instead "+
-			"handled by 'New' or 'Var'.",
-		name,
-	)
-}
-
-func DeprecatedTag(migrationPlan string) Taggable {
-	return &deprecatedTag{migrationPlan: migrationPlan}
-}
+var DeprecatedTag = flagtags.DeprecatedTag
```

Comment:
to remove one level of indirection, could we reference the `flagtags.*` members in the `flag` package and remove these references?